src/components/atom_word_cards_frame_parts/index.refresh-button.tsx

```diff
@@ -2,24 +2,21 @@ import { FC, useCallback } from 'react'
 import StyledCloudRefresher from '@/atoms/StyledCloudRefresher'
 import { useSemesters } from '@/hooks/semesters/use-semesters.hook'
 import { useWords } from '@/hooks/words/use-words.hook'
-import { useSetRecoilState } from 'recoil'
-import { preferenceState } from '@/recoil/preferences/preference.state'
-import { getPreferenceApi } from '@/api/preferences/get-preferences.api'
+import { usePreference } from '@/hooks/preference/use-preference.hook'
 
 const WordCardsFrameRefreshButtonPart: FC = () => {
   const [, getWords] = useWords()
   const getSemesters = useSemesters()
-  const setPreferenceState = useSetRecoilState(preferenceState)
+  const getPreference = usePreference()
 
   const onClickRefresh = useCallback(async () => {
     const semesters = await getSemesters()
     if (!semesters.latestSemesterCode) return
 
     await getWords({ semester: semesters.latestSemesterCode })
 
-    const [data] = await getPreferenceApi()
-    setPreferenceState(data)
-  }, [getWords, getSemesters, setPreferenceState])
+    getPreference()
```

Comment:
await `getPreference()`

Reply:
Handled in https://github.com/ajktown/wordnote/pull/101/commits/930dc49a3a2b5a7ccc76c959b258b57393831ed7